app/javascript/app/pages/mitigation/mitigation-actions/mitigation-actions-component.jsx

```diff
@@ -1,15 +1,120 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import SectionTitle from 'components/section-title';
+import TabSwitcher from 'components/tab-switcher';
+import GHGInventoryProvider from 'providers/ghg-inventory-provider';
+import { NoContent, Loading, Table } from 'cw-components';
+
 import styles from './mitigation-actions-styles.scss';
 
-class Nav extends PureComponent {
+const ALL_ACTIONS_KEY = 'allActions';
+const WITH_QUANTIFIED_EFFECTS_KEY = 'quentifiedEffects';
+
+class GHGInventory extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tabs: [
+        {
+          name: 'ALL ACTIONS',
+          value: ALL_ACTIONS_KEY,
+          component: <Loading height="660" />
+        },
+        {
+          name: 'WITH QUANTIFIED EFFECTS',
+          value: WITH_QUANTIFIED_EFFECTS_KEY,
+          disabled: true,
+          component: <Loading height="660" />
+        }
+      ]
+    };
+  }
+
+  static getDerivedStateFromProps(props, state) {
+    if (props.tableData.data) {
+      const activeTabValue = props.activeTabValue || state.tabs[0].value;
+      return {
+        ...state,
+        tabs: state.tabs.map(tab => {
+          if (tab.value === activeTabValue) {
+            const hasContent = props.tableData.data &&
+              props.tableData.data.length > 0;
+            return {
+              ...tab,
+              component: hasContent
+                ? (
+                  <Table
+                    horizontalScroll
+                    tableHeight={660}
+                    setRowsHeight={() => 120}
+                    hasColumnSelect={false}
+                    {...props.tableData}
+                  />
+)
+                : (
+                  <NoContent
+                    minHeight={330}
+                    message={
+                    props.searchFilter
+                      ? 'No data found with this search'
+                      : 'No data available'
+                  }
+                  />
+)
+            };
+          }
+          return tab;
+        })
+      };
+    }
+    return state;
+  }
+
+  handleTabChange = ({ value }) => {
+    const { updateQueryParam, query } = this.props;
+    updateQueryParam({ query: { ...query, tab: value } });
+  };
+
+  handleFilterChange = value => {
+    const { updateQueryParam, query } = this.props;
+    updateQueryParam({ query: { ...query, search: value } });
+  };
+
   render() {
-    return <p className={styles.text}>The mitigation actions section</p>;
+    return (
+      <div className={styles.row}>
+        <SectionTitle title="GHG Inventory Improvement Programme" />
+        <TabSwitcher
+          disabled
```

Comment:
I think this is not needed as the disabled option is connected to the specific tab and it's specified in tabs array

Reply:
Good catch, fixed.